src/cart/crt_iv.c

```diff
@@ -2888,9 +2867,7 @@ bulk_update_transfer_done_aux(const struct crt_bulk_cb_info *info)
 			D_GOTO(exit, rc);
 		}
 
-		output->rc = iv_ops->ivo_on_put(ivns_internal,
-						&cb_info->buc_iv_value,
-						cb_info->buc_user_priv);
+		iv_ops->ivo_on_put(ivns_internal, &cb_info->buc_iv_value, cb_info->buc_user_priv);
 
```

Comment:
output->rc might end up being unset in this branch, need to set it to DER_SUCCESS either here or after line 2800 to be safe

Reply:
I did wonder about that and had that logic in a earlier version of this PR but DER_SUCCCESS is zero and the memory should be zeroed already?  Unless there's something that's already there that needs to be overwritten.

Reply:
Done.